app/assets/stylesheets/forms.scss

```diff
@@ -0,0 +1,118 @@
+// Form Elements
```

Comment:
We can't do this because this styling will be applied to EVERY element on the page. So every form input, every label, every checkbox and fieldset will be affected by this, not only these in onboarding.

You can either add a parent selectors like
```css
.onboarding-steps {
  fieldset {
    ...
  }
  ...
}
```

OR you can update your code with classnames, for example:

```css
// fieldset {
.onobarding-fieldset {
  ...
}
```

Reply:
@ludwiczakpawel noted - thanks! And FYI, I know that most of what is here will be overridden with Crayons' new form elements, but I think we can resolve that in a future PR.

Reply:
definitely 👍